src/UmbracoIdentity/IdentityEnabledMembersMembershipProvider.cs

```diff
@@ -49,6 +49,9 @@ public new string HashPasswordForStorage(string password)
 
         public new bool VerifyPassword(string password, string hashedPassword)
         {
+            if (hashedPassword == null)
```

Comment:
won't this basically bypass password security by just entering a blank password?